tests/common.sh

```diff
@@ -50,6 +50,18 @@ function wait_for_ping_connectivity {
     done
 }
 
+function wait_for_connectivity {
+    COUNTER=0
+    while [ $COUNTER -lt 40 ]; do
+        if echo "wait_for_connectivity" | nsenter --preserve-credentials -U -n --target=$1 ncat -v $2 $3; then
+            break
+        else
+            sleep 0.5
+        fi
+        let COUNTER=COUNTER+1
+    done
```

Comment:
exit with non-zero on error?

Reply:
No not really. Error should be resolved by code which calls it. This just waits for connectivity or times out. 

It is consistent with other waits.